execution_engine_testing/tests/src/test/contract_api/dictionary.rs

```diff
@@ -649,3 +650,19 @@ fn should_query_dictionary_items_with_test_builder() {
         assert_eq!(value, dictionary::DEFAULT_DICTIONARY_VALUE);
     }
 }
+
+#[ignore]
+#[test]
+fn should_be_able_to_perform_dictionary_read() {
```

Comment:
there should also be a test for normal read, which should also work now that is_readable is true for Key::Dictionary

Reply:
Added a test for normal read in https://github.com/casper-network/casper-node/pull/3161/commits/862d2b060f8391957da23d76025c2b8a6fd92c5f